skeleton-typescript-asp.net5/src/skeleton-navigation-typescript-vs/build/tasks/watch.js

```diff
@@ -8,8 +8,8 @@ function reportChange(event) {
 }
 
 gulp.task('watch', function() {
-  gulp.watch(paths.source, ['build-system']).on('change', reportChange);
-  gulp.watch(paths.html, ['build-html']).on('change', reportChange);
-  gulp.watch(paths.css, ['build-css']).on('change', reportChange);
+  gulp.watch(paths.source, ['build']).on('change', reportChange);
+  gulp.watch(paths.html, ['build']).on('change', reportChange);
```

Comment:
Is this also required for html & css? I've only had the issue with ts files.

Also, doing a clean & rebuilding ts/html/css files every time a file is changed seems to hammer VS. I suspect the clean is responsible since running gulp clean lags VS by a few seconds while it reorganizes its project files. This could just be a "my PC problem" however.